notebooks/lesson10_clusters.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<ul><li>"Learning objectives" for consistency in other lessons. in general i only capitalise the first letter in headings.</li><li>i would write "In this lesson we will have a look at distance and similarity measures to analyse data." (the two applications come later)</li><li>put the "k-nearest neighbour" and "k-means" sentences in a bullet list?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>